ui/src/Components/NavBar.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import TypoGraphy from '@material-ui/core/Typography'
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar'
+
+
+
+function NavBar(props) {
+
+    return (
+        <AppBar position="static">
+            <Toolbar>
+                <TypoGraphy variant="title"
```

Comment:
Install "Pretty Formatter" and format all the code to avoid merge conflicts in the future.  Let's have a standard formatter for everyone.

Reply:
done.